frontend/public/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -50,10 +51,12 @@ export const OperatorHubSubscribeForm = withFormState((props: OperatorHubSubscri
   }
 
   const {provider, channels = [], packageName} = props.packageManifest.data.status;
+  const srcProvider = _.get(props.packageManifest.data, 'metadata.labels.opsrc-provider');
+  const providerType = getOperatorProviderType(props.packageManifest.data);
 
   const submit = () => {
     const operatorGroupNamespace = props.operatorGroup.data.find(og => og.metadata.name === props.formState().target).metadata.namespace;
-    const OPERATOR_HUB_CSC_NAME = `${OPERATOR_HUB_CSC_BASE}-${operatorGroupNamespace}`;
+    const OPERATOR_HUB_CSC_NAME = `${OPERATOR_HUB_CSC_BASE}-${srcProvider}`;
```

Comment:
@spadgett I had to change the name of the catalogsourceconfig to be a function of the provider type so that we have one catalogsourceconfig per provider type as intended.